design/ceph/ceph-nfs-ganesha.md

```diff
@@ -213,6 +213,21 @@ The following directories should **not** be shared between SSSD and other contai
 - `/run/dbus`: using the DBus instance from the sidecar caused SSSD errors in testing. SSSD only
   uses DBus for internal communications and creates its own socket as needed.
 
+### Metrics
+Each NFS-Ganesha server instance exports its stats via DBus. When Prometheus
+monitoring is enable on a k8s cluster, run a sidecar container which will
+provide an end-point for Prometheus metrics scraping. This sidecar container
+must share the `/run/dbus` directory in order to communicate with the DBus
+daemon via UNIX domain socket within the same pod (similar to way NFS-Ganesha's
+`ganesha_stats` utility communicates with the server).
+
+Exported metrics should include the following details:
+- Exports
+- Connected clients
+- Operations counter per export (for each sub-protocol)
```

Comment:
What is `sub-protocol` in this context? Could we define that here?

Reply:
Currently, `NFSv3, NFSv4.0, NFSv4.1, NFSv4.2`. That is, report information at fine-grained NFS protocol granularity.